otterdog/eclipse-tractusx.jsonnet

```diff
@@ -1517,5 +1517,22 @@ orgs.newOrg('eclipse-tractusx') {
         default_workflow_permissions: "write",
       },
     },
+    orgs.newRepo('wallet-dim-stub') {
+      description: "Wallet DIM stub application",
```

Comment:
Maybe "Stub application for DIM Wallet" tells a bit more

Reply:
Addressed with: https://github.com/eclipse-tractusx/.eclipsefdn/pull/98/commits/e5bb6ba7e3fb1755b64d25973d53eb9e116cda27